packages/graphback-runtime/src/data/KnexDBDataProvider.ts

```diff
@@ -45,8 +43,7 @@ export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements G
     }
 
     // tslint:disable-next-line: no-reserved-keywords
-    public async delete(inputType: InputModelTypeContext,  id: string): Promise<string> {
-        const name = getTableName(inputType.name)
+    public async delete(name: string, id: string, data?: Type): Promise<string> {
```

Comment:
Is `data` needed here? I think it is unlikely to be used in `delete`

Reply:
Delete uses data for conflict resolution. In offix spec we need data for version (or any marker) to determine if operation triggered conflict.